### ISSUE TITLE
Draw loss function graph

### ISSUE THREAD

**Issue by jas-nat** [facebookresearch/detectron2#560]
Can we draw the graph of total loss while training the model?

**Comment by rsadiq** [comment]
You can use tensorboard to visualize all graphs during training as explained in detectron2 colab notebook section.
[https://colab.research.google.com/drive/16jcaJoc6bCFAQ96jDe2HwtXj7BMD_-m5#scrollTo=hBXeH8UXFcqU&line=3&uniqifier=1](url)

 Also the loss, accuracy and total_loss is written in a json file ```metrics.json
``` along with model_xxx.pth in snapshot directory, After training you can extract the desired values from the  json file.

**Comment by jas-nat** [comment]
Can I use the tensorboard locally?

**Comment by ppwwyyxx** [comment]
Please see @rsadiq 's detailed answer.

You can use it locally by using `tensorboard --logdir some_dir`. Please refer to its documentation for usage.
